docs/data/toolpad/connecting-to-datasources/connections.md

```diff
@@ -4,23 +4,23 @@
 
 To create a new connection instance, press the **+** button in the Explorer menu on the left side of the interface:
 
-<img src="/static/toolpad/connection.png" alt="Add connection" width="200px" />
+<img src="/static/toolpad/connection.png" alt="Add connection" width="196px" />
```

Comment:
> did you fetched dimensions manually

@bytasv I have done the change manually, using the Chrome Dev Tools. https://www.notion.so/mui-org/Images-solution-30e78dbf24744ed5b570027656139092 is exploring on how to automate this.

> that large images seems to get downsized anyways because we have a limit on our content width

With the changes, these large images now use the whole space available. I assume it's a win?

> IMO when everything is downsized it looks a little bit bitter.

"bitter" as bitter taste, or better? So I guess we are talking about how to handle images that can fit inside the body width. Should we downsize them or not?

The advantage of having them 1:1, is I think: readability, assuming the interface we screenshot is already readable, by keeping it 1:1 we guarantee that nothing will be too small to read. Fidelity, maybe it's easier to orient oneself when the scale is the same. No artifacts when browser resizing, the resizing is either 1:1 or 2:1.
Now, feel free to revert, I have no real preferences.

>  I was thinking that maybe it would be better (if possible) to use some modal to show original size images when clicked? (i.e. how medium does it)

Agree, this could be a great addition. like https://stripe.com/docs/payments/link/accept-a-payment?platform=web#web-create-intent.

Reply:
> we don't get much benefit of images that are larger than ~780px.

Agree, but it still feel better by using all the available space, right 😁. 

> For me it seems a bit disproportionate. So rather than try to fit some images 1:1, idea was to downscale so they ALL look kinda similar and do not standout with their scale.

Sounds fair. So from my points above. We can discard point B. and only enforce A. and C. Does it with you?

> Additionally, IMO it would be really good (and maybe not too difficult?) to add "image zoom" on click?
I could ping core team and check how I could implement this in a current doc infra

Two extra thoughts: 1. the core team doesn't own this one, the docs infra team does. 2. the docs infra team might actually need a maintainer to represent the interests of Toolpad as we grow.

Reply:
Alright, let's leave it as is, I don't have strong preferences 👍 

Who are in fact docs infra team? 🤔 I assume it's a virtual setup since I couldn't find any info. So far I think only Jun has been replying to request